frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -975,6 +993,7 @@ function ReaderFooter:textOptionTitles(option)
         frontlight_warmth = T(_("Warmth level (%1)"), symbol_prefix[symbol].frontlight_warmth),
         mem_usage = T(_("KOReader memory usage (%1)"), symbol_prefix[symbol].mem_usage),
         wifi_status = T(_("Wi-Fi status (%1)"), symbol_prefix[symbol].wifi_status),
+        book_authors = _("Book authors"),
```

Comment:
Just "Author" should suffice, unlike with title, there is no confusion to be had anywhere. By that i mean, title could mean the book's title or the title of a chapter, same does not happen to the author. Also, keep it singular (even though some documents might have multiple ones). The rule is singular, exception is plural.